README.md

```diff
@@ -12,20 +12,48 @@ INTRODUCTION
 ------------
 
 The Devportal module allows you to expose API reference documentation by
-uploading Swagger/OpenAPI files - as well as create, edit, import and publish
+publishing Swagger/OpenAPI files - as well as create, edit, import and publish
 conceptual documentation on your Drupal site.
 
  * For a full description of the module, visit the project page:
    https://www.drupal.org/project/devportal
-
+   
```

Comment:
This change is unneeded (and lines ending with whitespace are usually treated bogus).